ci/cloudbuild/build.sh

```diff
@@ -264,8 +269,20 @@ fi
 
 # Uses Google Cloud build to run the specified build.
 io::log_h1 "Starting cloud build: ${BUILD_NAME}"
-project="${PROJECT_FLAG:-$(gcloud config get-value project)}"
-account="$(gcloud config get-value account)"
+project="${PROJECT_FLAG:-$(gcloud config get-value project 2>/dev/null)}"
+account="$(gcloud config get-value account 2>/dev/null)"
+# The cloudbuild.yaml file expects certain "secrets" to be present in the
+# project's "Secret Manager". This is true for our main production project, but
+# for personal projects we may need to create them (with empty strings).
+if [[ "${project}" != "cloud-cpp-testing-resources" ]]; then
+  for secret in "CODECOV_TOKEN" "LOG_LINKER_PAT"; do
+    if ! gcloud --project "${project}" secrets describe "${secret}" >/dev/null; then
+      io::log_yellow "Adding missing secret ${secret} to ${project}"
+      gcloud --project "${project}" secrets create "${secret}" --data-file=- \
+        </dev/null
```

Comment:
Just `--data-file=/dev/null`?

Reply:
I'm embarrassed that my chain of refactoring stopped just short of the clear expression that you suggest here. Changing... oh, to my surprise that doesn't work. Well, it creates a secret but without any "versions" (i.e., no values associated w/ the secret key), and GCB requires at least one version, so this doesn't work. However, I did change this line to read `echo | gcloud ...`, which is still shorter and clearer than what I had before.

Reply:
> Changing... oh, to my surprise that doesn't work. Well, it creates a secret but without any "versions"

That indeed sounds wacky.